### PR TITLE
Test - Test Search location

### PR DESCRIPTION
**Pull request by at-ynguyen**
#### Related issues (Required)
- Task #192 

#### Description (Required)
- [x] Unit test trigger search location result
- [x] Unit test get location detail

### CODE REVIEW

File: app/src/test/java/vn/asiantech/way/search/SearchViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    fun initTests() {
+        MockitoAnnotations.initMocks(this)
+        viewModel = SearchViewModel(wayRepository, localRepository)
+    }
+
+    @Test
+    fun `Given place id - When call get location detail - Then return result location`() {
+        /* Given */
+
+        val resultPlace = ResultPlaceDetail(WayLocation("abcde", "abcde", "abcde", "abcde"))
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/search/SearchViewModelTest.kt]: change this line to `val resultPlace = ResultPlaceDetail(WayLocation("id", "placeId", "name", "formattedAddress"))`

File: app/src/test/java/vn/asiantech/way/search/SearchViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+
+        viewModel.triggerSearchLocationResult(input, sensor).subscribe(triggerSearchLocationResultTest)
+        triggerSearchLocationResultTest.assertValue { it == autoCompleteResult.predictions }
+
+
+    }
+
+}
+
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/search/SearchViewModelTest.kt]: remove this line

File: app/src/test/java/vn/asiantech/way/search/SearchViewModelTest.kt [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        /* Then */
+
+        viewModel.getLocationDetail(resultPlace.result.placeId!!).subscribe(getLocationDetailTest)
+        getLocationDetailTest.assertValue { it == resultPlace.result }
+    }
+
+
+    @Test
+    fun `Given input ,language, sensor  - When call trigger search location result  - Then return list location `() {
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/search/SearchViewModelTest.kt]: change this line to:
`fun Given input, language, sensor - When call trigger search location result  - Then return list location () {`

**Comment by at-vinhhuynh** [comment]
@at-tienhoang 
I finish review this PR, please check it again.
thank you!

**Comment by at-dev-support-01** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please rebase to get rid of the merge commits in this PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">All test passed. (5/5)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**